src/scratch/UCERF3/FaultSystemRupSet.java

```diff
@@ -65,7 +65,7 @@
 	private String info;
 
 	private PlausibilityConfiguration plausibilityConfig;
-	private List<ClusterRupture> clusterRuptures;
+	protected List<ClusterRupture> clusterRuptures;
```

Comment:
should be able to just use getter/setters

Reply:
this was reverted too